### PR TITLE
Added twig code standard check using twigcs

### PR DESCRIPTION
**Pull request by woredeyonas**
I've added twigcs in the default `grumphp.yml.dist`

### CODE REVIEW

File: grumphp.yml.dist [hunk 1]
```diff
@@ -9,6 +9,7 @@ parameters:
     yamllint: ~
     composer: ~
     jsonlint: ~
+    twigcs: ~
```
Comment (skippednote) [inline grumphp.yml.dist:12]: Drupal core doesn't conform to the twigcs rules and would throw a lot of errors. We could exclude core and contrib to work around it. The other option, which I'm not completely in favour of, would be to create a custom ruleset that matches the way Drupal core formats twig files.

Comment (hussainweb) [reply grumphp.yml.dist:12]: We should add the `path` option set to `web/themes/custom`.

**Comment by hussainweb** [comment]
I'm thinking we should also add `friendsoftwig/twigcs` to composer.json's suggests list. We don't want to add it to `require` in case somebody can't use it for any reason.

File: grumphp.yml.dist (hunk 1 above)
Comment (skippednote) [reply grumphp.yml.dist:12]: @hussainweb That would not cover templates in custom/modules.

**Comment by hussainweb** [comment]
Closes #3

File: grumphp.yml.dist (hunk 1 above)
Comment (hussainweb) [reply grumphp.yml.dist:12]: @skippednote, sure, but unless we have a better solution, this is valuable at least. I feel we could go ahead with this now as default and figure out if grumphp can run tasks repeatedly.

Comment (skippednote) [reply grumphp.yml.dist:12]: @hussainweb My suggestion is that we set the path to `web` or `.` and ignore `contrib/*` and `core` .

Comment (hussainweb) [reply grumphp.yml.dist:12]: It looks like it's already possible. https://github.com/phpro/grumphp/blob/master/doc/tasks.md#run-the-same-task-twice-with-different-configuration

Comment (skippednote) [reply grumphp.yml.dist:12]: why run twice when we can handle it in a single run?
```suggestion
    twigcs: 
      path: '.'
      exclude:
        - core
        - contrib/*
```

Comment (hussainweb) [reply grumphp.yml.dist:12]: > My suggestion is that we set the path to web
@skippednote, the problem is that will include core and contribs. How would you ignore them?

I feel that running the task twice, once with modules and then with themes might work better.

Comment (hussainweb) [reply grumphp.yml.dist:12]: Ah, I see it has an option to exclude. That works too. We should list every possible `contrib` path, no?

Comment (skippednote) [reply grumphp.yml.dist:12]: >@skippednote, the problem is that will include core and contribs. How would you ignore them?
Check the example above, we can use `exclude` to ignore paths.

Comment (hussainweb) [reply grumphp.yml.dist:12]:
```suggestion
    twigcs:
      path: 'web'
      exclude:
        - core
        - modules/contrib
        - themes/contrib
        - profiles/contrib
```

Comment (hussainweb) [reply grumphp.yml.dist:12]: The paths are supposed to be relative from the [documentation](https://github.com/phpro/grumphp/blob/master/doc/tasks/twigcs.md).

> This option will specify which relative subfolders or files will be exclude to this task.

**Pull request merged**